providers/implementations/rands/seed_src_jitter.c

```diff
@@ -81,24 +83,32 @@ static size_t get_jitter_random_value(PROV_JITTER *s,
 {
     struct rand_data *jitter_ec = NULL;
     ssize_t result = 0;
+    size_t num_tries;
+
+    /* Retry intermittent failures, then give up */
+    for (num_tries = 0; num_tries < JITTER_MAX_NUM_TRIES; num_tries++) {
+        /* Allocate a fresh collector */
+        jitter_ec = jent_entropy_collector_alloc(0, JENT_FORCE_FIPS);
+        if (jitter_ec == NULL)
+            continue;
+
+        /* Do not use _safe API as per typical security policies */
```

Comment:
can you elaborate on the reasoning for this comment?

Reply:
@nhorman _safe API automatically increases OCR and ignores intermittent and permanent failures. Security policies for all publicly available jitter entropy sources thus do not permit using _safe() api call.

> the jent_read_entropy_safe of the master branch still cannot be used because it re-allocates the entropy collector if a health test failure is observed. Preventing re-init and preventing the oversampling rate from increasing would make the function FIPS compliant. I don't think that the function will be re-implemented upstream to make it FIPS compliant, because there is another function that is already FIPS compliant: jent_read_entropy. You should use jent_read_entropy.